plot-gui-lib/plot/private/gui/snip2d.rkt

```diff
@@ -15,9 +15,18 @@
          "worker-thread.rkt"
          "snip.rkt")
 
-(provide make-2d-plot-snip
-         plot-mouse-event-callback/c
-         2d-plot-snip+c%)
+(define plot-mouse-event-callback/c
+  (-> (is-a?/c snip%) (is-a?/c mouse-event%) (or/c real? #f) (or/c real? #f) any/c))
+(define 2d-plot-snip+c%
```

Comment:
Since this identifier is just the contract, the name `2d-plot-snip%/c` would be more idiomatic*.

`*` More like the `class/c` identifier in `https://github.com/racket/draw/blob/master/draw-lib/racket/draw/private/contract.rkt`